adafruit_avrprog.py

```diff
@@ -339,13 +378,15 @@ def read(self, addr, read_buffer):
             last_addr = read_addr
 
     #################### Low level
-    def _flash_word(self, addr, low, high):
+    def _flash_word(self, addr: int, low: int, high: int) -> None:
         self._transaction((0x40, addr >> 8, addr, low))
         self._transaction((0x48, addr >> 8, addr, high))
 
-    def _flash_page(self, page_buffer, page_addr, page_size):
+    def _flash_page(
+        self, page_buffer: bytearray, page_addr: int, page_size: int
+    ) -> None:
         page_addr //= 2  # address is by 'words' not bytes!
-        for i in range(page_size / 2):  # page indexed by words, not bytes
+        for i in range(floor(page_size / 2)):  # page indexed by words, not bytes
```

Comment:
range() is technically expecting an int and the division of `page_size / 2` productes a float (even if it's evenly divisible)
```
In[2]: 32 / 2
Out[2]: 16.0
```
My IDE warns about this at this line, but nothing in our pre-commit or pylint checks do.


To my eye it looks like it could use `//` to ensure the result of the division is an int but I was not 100% certain and I didn't want to cause a behavior change so I just removed floor and put it back to how it was for now.

Reply:
`// 2` should be fine here, or you could use `>> 1`.